lib/arrays.md

```diff
@@ -4,45 +4,53 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 ---
 
 # What is an Array?
+An Array is a Ruby data type that stores an ordered list of values.  
+An Array is a number-indexed list.  
+Values in a Array can be any type of object, including other Arrays.
 
-Replace me with your response...
 
 # What are some examples of information that would be Arrays as opposed to some other data type?
+Some examples of information that would be Arrays are a collection of values or list of data.
 
-Replace me with your response...
 
 # What is one way, using Ruby, to retrieve the 6th element in an Array? How about the 8th element? What happens if you try to retrieve the value of the _9999th_ element (or some element that doesn't exist in the array)?
+We can retrieve the 6th element in an Array by using the #[5] method, the 8th element using the #[7] method.
+If we try to retrieve the value of an element that doesn't exist in our Array we will get => nil as a result.
 
-Replace me with your response...
 
 # The previous question asks about finding, for example, the 6th element in an Array. Is it possible to find the **-6th** (Notice the negative symbol!) element in an Array? What does that even mean?
+Assuming the Array contains at least 6 elements, we can find the -6th element.  The -6th element would be the 6th to last.
 
-Replace me with your response...
 
 # How would you perform an operation on every element inside an Array?
+Here we can perform an operation on every element inside an Array by using the Enumeral method '.map'.
+For example, array = [1,2,3,4,5,"Fiona"].map{ |element| element * 3} => [3, 6, 9, 12, 15, "FionaFionaFiona"]
 
-Replace me with your response...
 
 # How do you add elements to an Array?
+We can add elements to an Array using the shovel '<<' operator or '.push' method.
 
-Replace me with your response...
 
 # Given the Array `["Laura", "Fiona", "Tori"]`, how would you replace `"Fiona"` with `"Florence"` so that you end up with `["Laura", "Florence", "Tori"]`?
+Although probably not the best example, we can use the '.map { |item| block} method where ["Laura", "Fiona", "Tori"].map{ |item| item.gsub("Fiona","Florence")} => ["Laura", "Florence", "Tori"]
```

Comment:
I could have just used the '[]=' method.  I was hung up on wanting to use the '.gsub' method.
names = ["Laura", "Fiona", "Tori"]
names[1] = "Florence"
names

This results in => ["Laura", "Florence", "Tori"]